### ISSUE TITLE
Depot Install Error: Wrong Number of Arguments

### ISSUE THREAD

**Issue by ColonelPanics** [alces-software/gridware#19]
When trying to install any depot, the following error is seen:
```
[moosepowered@login1(mycluster) ~]$ alces gridware depot install benchmark
Installing depot: benchmark

 > Initializing depot: benchmark
      Initialize ... OK

error: wrong number of arguments (2 for 3). Use --trace to view backtrace
```
Trace:
```
[moosepowered@login1(mycluster) ~]$ alces gridware depot install cfd --trace
Installing depot: cfd

 > Initializing depot: cfd
      Initialize ... OK

/opt/clusterware/opt/gridware/lib/alces/packager/archive_importer.rb:44:in `initialize': wrong number of arguments (2 for 3) (ArgumentError)
    from /opt/clusterware/opt/gridware/lib/alces/packager/archive_importer.rb:34:in `new'
    from /opt/clusterware/opt/gridware/lib/alces/packager/archive_importer.rb:34:in `import'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:262:in `block (2 levels) in install'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:258:in `each'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:258:in `block in install'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/dm-core-1.2.1/lib/dm-core.rb:263:in `block in repository'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/dm-core-1.2.1/lib/dm-core/repository.rb:114:in `scope'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/dm-core-1.2.1/lib/dm-core.rb:263:in `repository'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:256:in `install'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:52:in `block in handle'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:42:in `each'
    from /opt/clusterware/opt/gridware/lib/alces/packager/depot_handler.rb:42:in `handle'
    from /opt/clusterware/opt/gridware/lib/alces/packager/handler.rb:153:in `depot'
    from /opt/clusterware/opt/gridware/lib/alces/packager/handler.rb:82:in `block in handle_action'
    from /opt/clusterware/opt/ruby/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler.rb:245:in `block in with_clean_env'
    from /opt/clusterware/opt/ruby/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler.rb:232:in `with_original_env'
    from /opt/clusterware/opt/ruby/lib/ruby/gems/2.2.0/gems/bundler-1.10.6/lib/bundler.rb:238:in `with_clean_env'
    from /opt/clusterware/opt/gridware/lib/alces/packager/handler.rb:77:in `handle_action'
    from /opt/clusterware/opt/gridware/lib/alces/packager/handler.rb:62:in `method_missing'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:179:in `call'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:68:in `run!'
    from /opt/clusterware/opt/gridware/vendor/ruby/2.2.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
    from /opt/clusterware/libexec/actions/gridware:77:in `<main>'
```

This occurred on a 2017.2rc2 Flight Compute setup on AWS. Previous to attempting a depot install, a few applications were installed (one of them using the `-g`/`--global` install option)

**Comment by jamesremuscat** [comment]
Fixed in https://github.com/alces-software/gridware/commit/d44f82b56ad09a98a239b83b8793b80ebcee83fe.

**Comment by jamesremuscat** [comment]
Released as `gridware-20171121-cw1_9`; merged into the `release/1.9` branch of Clusterware.
